main.js

```diff
@@ -0,0 +1,108 @@
+var sides = [
+  "Miso Glazed Carrots",
+  "Coleslaw",
+  "Garden Salad",
+  "Crispy Potatoes",
+  "Sweet Potato Tots",
+  "Coconut Rice",
+  "Caeser Salad",
+  "Shrimp Summer Rolls",
+  "Garlic Butter Mushrooms",
+  "Hush Puppies"
+]
+
+
+var mains = [
+  "Spaghetti and Meatballs",
+  "Pineapple Chicken",
+  "Shakshuka",
+  "Thai Yellow Curry",
+  "Bibimbap",
+  "Chicken Parmesean",
+  "Butternut Squash Soup",
+  "BBQ Chicken Burgers",
+  "Ramen",
+  "Empanadas",
+  "Chicken Fried Rice",
+  "Sheet Pan Fajitas",
+  "Margarita Pizza"
+];
+
+
+var desserts = [
+  "Apple Pie",
+  "Lemon Meringue Pie",
+  "Black Forest Cake",
+  "Banana Bread",
+  "Peach Cobbler",
+  "Cheesecake",
+  "Funfetti Cake",
+  "Baklava",
+  "Flan",
+  "Macarons",
+  "Macaroons",
+  "Chocolate Cupcakes",
+  "Pavlova",
+  "Pumpkin Pie",
+  "Key Lime Pie",
+  "Tart Tatin",
+  "Croissants",
+  "Eclairs"
+];
+
+var currentFood;
+
+var mealTypeForm = document.getElementById("type-form");
+var inputValues = document.getElementsByName("meal-type");
+var randomDishSection = document.getElementById("random-dish");
+var clearButton = document.getElementById("clear-button");
+var cookpotImage = document.getElementById("cookpot-icon");
+
+console.log(clearButton)
+
+mealTypeForm.addEventListener('submit', getInputValue);
+clearButton.addEventListener('click', clearView);
+
+
+function getRandomIndex(array) {
+  return Math.floor(Math.random() * array.length);
+};
+
+function getInputValue() {
+  event.preventDefault();
+  for (var i =0; i < inputValues.length; i++) {
+    if (inputValues[i].checked) {
+      currentFood = inputValues[i].value;
+    }
+  }
```

Comment:
This might be a good opportunity to refactor this and practice using your array prototype methods!